tests/test_pedigree.py

```diff
@@ -174,3 +175,153 @@ def test_pedigree_sanity_checks(self):
 
         with pytest.raises(ValueError):
             ped.get_times(individual=ped.individual)
+
+
+def simulate_pedigree(
+    n_founders=8,
+    n_children_prob=None,
+    n_generations=3,
+    random_sex=True,
+) -> tskit.IndividualTable:
```

Comment:
We need to control randomness here, so we want to have a ``random_seed`` argument. Since this is just a testing method we can make it default to some value (``random_seed=42``, say)